gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html

```diff
@@ -96,5 +96,25 @@
     </div>
   </mat-card>
 
+  <ng-container *ngIf="!is0AuthClient">
```

Comment:
we only have 2 possibilities here? I wonder if a switch wouldn't be cleaner.
or a `*ngIf="applicationType === 'oauth2' "`

Reply:
Yes, we have two types: SIMPLE, NATIVE
Done